go/vt/tabletserver/logstats.go

```diff
@@ -50,7 +50,9 @@ type LogStats struct {
 	Error                *TabletError
 }
 
-func newLogStats(methodName string, ctx context.Context) *LogStats {
+// NewLogStats constructs a new LogStats with supplied Method and ctx field values, and the StartTime field set to
```

Comment:
Exposing the previously-private `newLogStats(...)` function, as per discussion with Alain and Sugu in Slack, in order to facilitate creating fake `LogStats` records in unit testing.